scripts/tests/test_validate_docstrings.py

```diff
@@ -229,6 +229,27 @@ def good_imports(self):
         """
         pass
 
+    def no_returns(self):
+        """
+        Say hello and have no returns.
+        """
+        print("Hello World!")
```

Comment:
```suggestion
        pass
```
unnecessary, and not sure if this gets printed in the logs